bundles/binding/org.openhab.binding.insteonplm/src/main/resources/device_features.xml

```diff
@@ -18,6 +18,15 @@
 	<poll-handler>NoPollHandler</poll-handler>
 </feature>
 
+<feature name="RampOnOff">
+	<message-dispatcher>DefaultDispatcher</message-dispatcher>
+	<message-handler cmd="0x19">RampOnOffSwitchHandler</message-handler>
```

Comment:
Please see my earlier answer regarding 0x19 and also the discussion in the linked feature request. Message handlers are looked up using a key. Sometimes the key is cmd1 and sometimes, as in this case, it is not. Take a look at the the DIRECT ACK handling in the MessageDispatcher that I mentioned earlier. There is a comment that says cmd1 is useless for DIRECT ACK, but this is not always the case. In this scenario, the lookup key is changed to 0x19 but cmd1 remains a valid value (0x34 or 0x35). The XML message-handler entries for 0x34 and 0x35 could actually be removed from device_features.xml although the feature definition would then break if the DIRECT ACK handling is fixed in the future. In any case, the handler instance must process the cmd1 values of 0x34 and 0x35. If this is not clear, maybe @berndpfrommer can describe it better.